ios/RCTMGL/RCTMGLMapView.m

```diff
@@ -78,6 +93,30 @@ - (void)setReactUserTrackingMode:(int)reactUserTrackingMode
     [self setUserTrackingMode:(NSUInteger)_reactUserTrackingMode animated:_animated];
 }
 
+#pragma mark - methods
+
+- (CLLocationDistance)getMetersPerPixelAtLatitude:(double)latitude withZoom:(double)zoomLevel
```

Comment:
The usual case would be to convert from pixels to meters at the current zoom level. Maybe make the zoom level optional?